docs/src/simulation_tips.md

```diff
@@ -200,7 +201,7 @@ to achieve this
 
 ### Arrays in GPUs are usually different from arrays in CPUs
 
-On the CPU Oceananigans.jl uses regular `Array`s, but on the GPU it has to use `CuArray`s
+Oceananigans.jl uses regular `Array`s when running on the CPU, but on the GPU it has to use `CuArray`s
 from the CUDA.jl package. While deep down both are arrays, their implementations are different
 and both can behave very differently. Something to keep in mind when working with `CuArray`s is that you do not want to
 access elements of a `CuArray` outside of a kernel. Doing so invokes scalar operations
```

Comment:
Is there something we can link to that explains what a GPU kernel is in simple terms? I couldn't find anything except for [this page](https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#kernels). I guess that's an okay reference, but I was hoping for something easier for a novice to understand.

Reply:
The CUDA.jl docs simply state "Kernels are functions that are executed in a massively parallel fashion..."

https://cuda.juliagpu.org/stable/usage/overview/#Kernel-programming-with-@cuda

Perhaps we can define a kernel as a "low-level function that executes in massively parallel fashion on the GPU"?

Reply:
Yes, but I was thinking of linking something, instead of explaining it in the text, so as to keep the discussion high level. Maybe we can link both urls for now and change it if we find something easier down the line.

Reply:
But why linking versus explaining makes this discussion high or low level?

Reply:
OK, perhaps this was resolved already? Ignore my "noise". :)

Reply:
I believed it's solved. But thnaks :)